openlibrary/plugins/openlibrary/tests/test_home.py

```diff
@@ -180,13 +179,8 @@ def test_urls(self, render_template):
         assert self.link_count(self.render(book)) == 2
 
         del book['daisy_url']
-        assert 'Borrow this book' in self.render(book)
-        assert book['overdrive_url'] in self.render(book)
```

Comment:
Interesting point. We want Overdrive to vanish from the code, and removing any assertions is weaker than asserting that *it is not there*. OTOH, that leaves an artifact from the past sitting in the current code.

Maybe this is a two-commit refactor: 1) Add the not-there assertion to drive out the Overdrive reference, and afterwards, 2) Remove the legacy reference.